third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -3170,6 +3234,16 @@ func flattenClusterTelemetry(c *containerBeta.ClusterTelemetry) []map[string]int
 	return result
 }
 
+func flattenDefaultSnatStatus(c *containerBeta.DefaultSnatStatus) []map[string]interface{} {
+	result := []map[string]interface{}{}
+	if c != nil {
```

Comment:
You may want to return disabled: true if c == nil, it helps `plan` make better decisions.  :)

Reply:
that makes the default value as `disabled: true`, right ?. What if i don't want to set the attribute if its not in the config ?

Reply:
Added tests for this attribute.